lld/test/ELF/bp-section-orderer-errs.s

```diff
@@ -0,0 +1,44 @@
+# REQUIRES: aarch64
+
+# RUN: llvm-mc -filetype=obj -triple=aarch64 %s -o %t
```

Comment:
`rm -rf %t && split-file %s %t && cd %t`

delete `-o /dev/null` for not ld.lld tests

See `2d756d9d4c89ac50404f942f2db2f4a402aa0e00`